CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
-# HEAD
+# 7.1.0
 
 ## Features
 
 * **css:** Add Notification bar component (#383)
 * **css:** Details button gets cursor:pointer when hovered (#367)
-* **css:** Add Metropolis as Firefox brand font (#386)
+* **css:** (breaking) Add Metropolis as Firefox brand font (#386)
```

Comment:
If we're calling something out as breaking then we should be doing a major version bump. If we think this is safe, then I think we can remove `(breaking)`.